lblod/frontend-gelinkt-notuleren
# Feat/download meeting part

### Overview
Added a way of downloading different parts of the meeting as HTML
https://www.figma.com/design/UVPMVe7O29nt7G1rRJ6uH9/installatievergadering?node-id=351-337&t=M0DRZAdacQNR1WNw-4
##### connected issues and PRs:
GN-4902


### Setup
None

### How to test/reproduce
Go to a meeting, click on the acties menu, and then on the "Onderdelen kopiëren en downloaden" you should be able to download all the meeting parts there

### Challenges/uncertainties
None



### Checks PR readiness
